crates/lovely-core/src/lib.rs

```diff
@@ -43,8 +43,11 @@ impl Lovely {
 
         let args = std::env::args().skip(1).collect::<Vec<_>>();
         let mut opts = Options::new(args.iter().map(String::as_str));
-    
+        #[cfg(target_os = "windows")]
         let mut mod_dir = dirs::config_dir().unwrap().join("Balatro\\Mods");
+        #[cfg(target_os = "macos")]
+        let mut mod_dir = dirs::config_dir().unwrap().join("Balatro/Mods");
+        
```

Comment:
Replacing these lines with 
```rust
let mut mod_dir = dirs::config_dir()
    .unwrap()
    .join("Balatro")
    .join("Mods");
```
should also work and won't require the conditional compilation flag.